Mapping application has been developed

mikhailkorzh opened the pull request.

mikhailkorzh committed:
Mapping application has been developed

mikhailkorzh committed:
pretty print code

Mak0 commented on src/main/java/com/company/cource/homeworks/homework14_mapper/mapping/GameEntityFactory.java:
```diff
@@ -0,0 +1,22 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+public class GameEntityFactory {
```
Зачем этот класс??

Mak0 commented on src/main/java/com/company/cource/homeworks/homework14_mapper/mapping/GameEntity.java:
```diff
@@ -0,0 +1,70 @@
+
+import java.util.Date;
+
+public class GameEntity {
```
Почему просто не назвать Game?

Mak0 commented on src/main/java/com/company/cource/homeworks/homework14_mapper/exceptions/NotFoundEntityException.java:
```diff
@@ -0,0 +1,7 @@
+package homeworks.homework14_mapper.exceptions;
+
+public class NotFoundEntityException extends Exception {
```
Очень сложное и не читаемое название. Лучше уж GameNotFoundException

Mak0 commented on src/main/java/com/company/cource/Exceptions/ScannerException.java:
```diff
@@ -0,0 +1,7 @@
+package Exceptions;
+
+public class ScannerException extends Exception {
```
Что это и зачем это тут?

Mak0 commented on src/main/java/com/company/cource/Exceptions/Exception2.java:
```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class Exception2 {
+    public static void main(String[] args) throws ScannerException {
```
что это и зачем это тут?

Mak0 commented on src/main/java/com/company/cource/homeworks/homework14_mapper/mapping/GamesFileDataMapper.java:
```diff
@@ -0,0 +1,61 @@
+        GameEntityFactory factory = new GameEntityFactory();
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader("D:\\JavaCoreCources\\src\\main\\java\\com\\company\\cource\\homeworks\\homework14_mapper\\newGameStore.txt"));
```
Путь к файлу должен быть относительным иначе на любом другом компе эта программа не отработает!

mikhailkorzh committed:
redundant files are deleted

mikhailkorzh committed:
1. class GameEntityFactory - the redundant class is removed (and related links)
2. GameEntity renamed to Game
3. NotFoundEntityException renamed to GameNotFoundException
4.  redundant class ScannerException  is removed (added to the project by mistake)
5.  redundant class class Exception2 is removed (added to the project by mistake)
6. the relative path to local Idea file is added ("./src/main/java/com/company/cource/homeworks/homework14_mapper/newGameStore.txt")
